CMakeLists.txt

```diff
@@ -1,74 +1,83 @@
-cmake_minimum_required(VERSION 3.13.4)
```

Comment:
Please decide for min version!

Reply:
@r0ost3r Please tell me what `cmake --version`s do you use on your local machines.
Is min_version 3.20 am option for you?

Reply:
cmake version 3.16.3